### ISSUE TITLE
Grants: side navigation isn't right

### ISSUE THREAD

**Issue by jalehman** [urbit/urbit.org#727]
Crawling the headers doesn't work right here—I don't want to see each grant as a navigable element. 

<img width="1390" alt="Screen Shot 2021-07-27 at 7 56 24 PM" src="https://user-images.githubusercontent.com/550978/127256498-7a012b3f-a805-4c08-9817-ac1236eabd2d.png">

The idea is to immediately tell the person that's just arrived to the page what it is that they can do here, and I think it should be these things:

1. Browse Grants
2. Submit a Proposal
3. View Gifts

Featured grants can be omitted because there's no useful action there.

Basically, I should just see three links between *Contributors of all types ... community mentors* and **Featured Grants**.

Also, let's remove the ~radbur quote. Sorry Tyler, but I think it doesn't quite fit with the theme.

**Comment by matildepark** [comment]
I tried this with a `noh3s` to only show the very, very top level headers and it looked pretty bare, but it's to taste I suppose.

**Comment by matildepark** [comment]
Fixed in https://github.com/urbit/urbit.org/commit/1d77c15312a5fe71bbbc202801ac11ce78d667be
